lib/preview/routes/proxy.js

```diff
@@ -0,0 +1,83 @@
+const { get } = require('../../../http');
+const { fetchContentMetadata } = require('../../../api/preview');
```

Comment:
```suggestion
const { get } = require('./http');
const { fetchContentMetadata } = require('./api/preview');
```

Based on some of the other files, you should be able to follow this format instead of the long relative trail